Backend/Services/RecipeService.cs

```diff
@@ -14,51 +14,58 @@ public RecipeService(IRecipeRepository repository)
     {
         _repository = repository;
     }
-    
+
```

Comment:
ignore the whitespace changes, I used auto formatter for this file and it just did that